daemon/mgr/cri.go

```diff
@@ -238,6 +278,26 @@ func makeContainerName(s *runtime.PodSandboxConfig, c *runtime.ContainerConfig)
 	}, nameDelimiter)
 }
 
+func parseContainerName(name string) (*runtime.ContainerMetadata, error) {
+	parts := strings.Split(name, nameDelimiter)
+	if len(parts) != 6 {
+		return nil, fmt.Errorf("failed to parse container name: %q", name)
```

Comment:
I think here we only throw the error, but have not told users the root cause of the error.

How about ` fmt.Errorf("failed to parse container name %q: name format must be in format blablabla", name)`